relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py

```diff
@@ -307,7 +318,36 @@ def install_dependencies(metadata):
             logger.warning("The binary " + executable + " was not found on the system, you must install it before installing " + metadata['name'])
             return False
       else:
```

Comment:
You should add a case in the same way that the "binary" type was defined. I am pretty sure we want to have typed dependencies and so, an "apt" and a "rpm" type of dependencies..